src/Home.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import deer from './icons/deer-black.svg';
+import './Home.css';
+
+export default function Home() {
```

Comment:
this was moved to pages/Home.tsx and components/Logo.tsx. I'll just merge this to get the rest of the changes, and update it later.